src/views/Home/props/footer.tsx

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable @typescript-eslint/no-empty-interface */
+import FacebookIcon from "images/components/footer/facebook.svg";
+import TwitterIcon from "images/components/footer/twitter.svg";
+import LinkedInIcon from "images/components/footer/linkedIn.svg";
+import InstagramIcon from "images/components/footer/instagram.svg";
+import YearIcon from "images/components/footer/2020.svg";
+interface SocialMediaProp {
+  key: string;
+  icon: string;
+  link: string;
+  label: string;
+}
```

Comment:
I understand why we have a `key` prop, but not entirely sure it's all that necessary. The `key` when we use the map function can simply be the `link` prop. Since the `link` will always be unique

Reply:
Again, since we are no longer using that key property, we don't need it anymore. Not a big deal, I think I'll give the code base a run through to have some consistency in how we store our props.